packages/espressocash_app/lib/ui/number_formatter.dart

```diff
@@ -5,8 +5,9 @@ import 'package:intl/intl.dart';
 extension NumberFormatterExt on String {
   Decimal toDecimalOrZero(Locale locale) {
     try {
+      final normalizedInput = replaceAll(',', '.');
       final formatter = NumberFormat.decimalPattern(locale.languageCode)..minimumFractionDigits = 2;
```

Comment:
Should it take into account `locale.languageCode` then if we're replacing delimiter?